apps/trace/tables.py

```diff
@@ -0,0 +1,60 @@
+import django_tables2 as tables
+from django.conf import settings
+from django.template.loader import get_template
+from django.urls import reverse
+
+from apps.generics import actions
+from apps.generics.actions import chip_action
+from apps.generics.tables import TimeAgoColumn
+from apps.trace.models import Trace
+
+
+def _chip_chatbot_url_factory(_, __, record, ___):
+    return reverse("chatbots:single_chatbot_home", args=[record.team.slug, record.experiment_id])
+
+
+def _chip_session_url_factory(_, __, record, ___):
+    return reverse(
+        "experiments:experiment_session_view",
```

Comment:
I see you're using the experiment url here but the chatbot url above. You might need to check the feature flag and the record type to determine where to redirect.